Apps/Admin/Server/appsettings.json

```
@@ -60,12 +60,12 @@
     "ContentSecurityPolicy": {
         "Base": "'self'",
         "DefaultSource": "'self'",
-        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha256-nYaPGPw9zQuQHVl09VRaW597Ut8F9Lf/VDNxGSQu3Sc=' 'sha384-/fIp3d9vqOYL5eP47pOEUYZ/h5dXL4fD9Rc4DWSuIqMZkUj+DA94qNELNYZm8VJS' 'sha384-G+v/7MQzsBBqsKVV3XKv8ThIDYIyXG6LJWqTPBrADbzQf/Ok8cTgjl0X+smak2c5' 'sha256-Tui7QoFlnLXkJCSl1/JvEZdIXTmBttnWNxzJpXomQjg=' 'sha256-vY0Cwh5hLv8sUQ61m3KQgfrqeAmcY7t6wJ2FAdYPNyU='",
+        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha512-uKQ39gEGiyUJl4AI6L+ekBdGKpGw4xJ55+xyJG7YFlJokPNYegn9KwQ3P8A7aFQAUtUsAQHep+d/lrGqrbPIDQ=='",
```

I think only the first 3 hashes should have been removed.  Relevant commits: 2a9a3314 (added the first 3 for Font Awesome) and 8a59885d (added the last 2 for Swagger).

```suggestion
        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha512-uKQ39gEGiyUJl4AI6L+ekBdGKpGw4xJ55+xyJG7YFlJokPNYegn9KwQ3P8A7aFQAUtUsAQHep+d/lrGqrbPIDQ==' 'sha256-Tui7QoFlnLXkJCSl1/JvEZdIXTmBttnWNxzJpXomQjg=' 'sha256-vY0Cwh5hLv8sUQ61m3KQgfrqeAmcY7t6wJ2FAdYPNyU='",
```